package/gluon-mesh-vpn-tunneldigger/luasrc/lib/gluon/upgrade/400-mesh-vpn-tunneldigger

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/lua
+
+local site = require 'gluon.site_config'
+local users = require 'gluon.users'
+local util = require 'gluon.util'
+
+local uci = require('simple-uci').cursor()
+
+
+-- Group for iptables rule
+users.add_group('gluon-tunneldigger', 900)
+
+local enabled = uci:get_first('tunneldigger', 'broker', 'enabled')
+if (tonumber(enabled) == 0) then
+  enabled = site.tunneldigger_mesh_vpn.enabled and 1 or 0
+end
```

Comment:
This condition looks wrong. Together with the changes below, I'd probably write something like:
```lua
local enabled = uci:get('tunneldigger', 'mesh_vpn', 'enabled')
if enabled == nil then
	enabled = uci:get_first('tunneldigger', 'broker', 'enabled')
end
if enabled == nil then
	enabled = site.tunneldigger_mesh_vpn.enabled or false
end
```
Note that the new *uci-simple* module can properly handle boolean values in set/section commands, you just need to make sure that the value is not *nil* (which is done here with the `or false`).